MainModule/Server/Core/HTTP.lua

```diff
@@ -76,7 +76,7 @@ return function(Vargs, GetEnv)
 						})
 					end
 				},
-				{
+				{ -- // Was this really a good idea? Since when should databases run code
```

Comment:
Was it a good idea? No. 

Only existed because I was too lazy to make a proper webpanel and was using Trello as one instead.... qq